support/remote_health_monitoring/remote-health-monitoring-data-usage-and-privacy.adoc

```diff
@@ -0,0 +1,39 @@
+[id="remote-health-monitoring-data-usage-and-privacy"]
+= Remote health monitoring data usage and privacy
+include::modules/common-attributes.adoc[]
+:context: remote-health-monitoring-data-usage-and-privacy
+
+toc::[]
+
+[id="telemetry-data-collection-notice_{context}"]
+== Telemetry data collection notice
+
+In connection with your use of this Red Hat offering, Red Hat may collect telemetry data about your use of the software. This data allows Red Hat to monitor the software in real time and to improve Red Hat offerings and support, including identifying, troubleshooting, and responding to issues that impact users. The data may also be used to enable you to track your entitlements to Red Hat subscriptions and take advantage of future Red Hat purchasing programs. It may also allow Red Hat to assist you in implementing upgrades to minimize service impact. The data may be shared internally within Red Hat to improve the user experience. If you are evaluating Red Hat software, the data will help Red Hat determine if you need assistance.
```

Comment:
>  This data allows Red Hat to monitor the software in real time and to improve Red Hat offerings and support

`real-time` should have a hyphen:

This data allows Red Hat to monitor the software in real-time and to improve Red Hat offerings and support.

Reply:
Thank you for highlighting that. I have now resolved this.

Reply:
@pneedle-rh thanks! LGTM

Reply:
@quarckster thank you!